package.json

```diff
@@ -67,6 +67,7 @@
     "@scure/base": "^1.1.3",
     "@types/debug": "^4.1.7",
     "debug": "^4.3.4",
+    "lodash": "^4.17.21",
```

Comment:
Not sure about using `lodash` just for the memoize function (especially because Lodash seems unmaintained).

Reply:
done